crates/bevy_ecs/src/schedule/mod.rs

```diff
@@ -28,6 +32,9 @@ use crate::{
 };
 use bevy_utils::HashMap;
 
+/// A container of [`Stage`]s set to be run in a certain order.
+///
+/// Since `Schedule` implements the [`Stage`] trait, it can be inserted into another schedule.
```

Comment:
A brief note about why you might want to do so would be useful here.

As I understand, this can be nice for creating complex nested control flows?